### PR TITLE
Move directives to misc module (third and last pack)

### PR DESCRIPTION
**Pull request by ger-benjamin**
For and close GSGMF-214

### CODE REVIEW

File: src/misc/decorate.js [hunk 1]
```diff
@@ -14,8 +14,6 @@ goog.require('ol.interaction.Interaction');
  *
  *      <input type="checkbox" ngModel="interaction.active" />
  *
- * See our live example: [../examples/interactiontoggle.html](../examples/interactiontoggle.html)
```
Comment (ger-benjamin) [inline src/misc/decorate.js]: Address comment from PR: https://github.com/camptocamp/ngeo/pull/3303#discussion_r158275716

File: docs/guidelines.md [hunk 2]
```diff
@@ -210,15 +210,15 @@ For example:
...
  * @ngInject
  * @ngdoc directive
  * @ngname ngeoControl
  */
-ngeo.controlDirective = function() {
+ngeo.misc.controlComponent.component_ = function() {
   // …
 };
-ngeo.module.directive('ngeoControl', ngeo.controlDirective);
+ngeo.misc.controlComponent.directive('ngeoMiscControl', ngeo.misc.controlComponent.component_);
```
Comment (gberaudo) [inline docs/guidelines.md]: I think you should keep 'ngeoControl'.

Comment (ger-benjamin) [reply docs/guidelines.md]: Yes, thanks, I don't know why I've do that

Comment (gberaudo) [reply docs/guidelines.md]: Otherwise LGTM.

Comment (ger-benjamin) [reply docs/guidelines.md]: Ahhh... but it's the guidlines. Does it matter ?
Perhaps it's even better to remove them...

Comment (gberaudo) [reply docs/guidelines.md]: The guideline describes how to deal with the Closure compiler symbols but says nothing about renaming the names used in Angular.

Comment (gberaudo) [reply docs/guidelines.md]: Ah, I understand what you mean now! ^^
OK, we can merge as it is.

**Comment by gberaudo** [comment]
@pfirpfel, if you have some time can you merge this PR tomorrow? Thanks.

**Pull request merged**